instrumentation/aws-sdk/aws-sdk-1.11/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v1_11/DynamoDbAttributesExtractor.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.awssdk.v1_11;
+
+import com.amazonaws.Request;
+import com.amazonaws.Response;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.AttributesBuilder;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.instrumentation.api.instrumenter.AttributesExtractor;
+import io.opentelemetry.instrumentation.api.internal.AttributesExtractorUtil;
+import java.util.Collections;
+import java.util.List;
+import javax.annotation.Nullable;
+
+public class DynamoDbAttributesExtractor implements AttributesExtractor<Request<?>, Response<?>> {
+
+  private static final AttributeKey<String> DB_SYSTEM = AttributeKey.stringKey("db.system");
```

Comment:
usually we add a comment about where this value was copied from like in
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/4f946640418a305e96dcd1b726ce307f930cbccc/instrumentation/aws-lambda/aws-lambda-events-2.2/library/src/main/java/io/opentelemetry/instrumentation/awslambdaevents/v2_2/internal/ApiGatewayProxyAttributesExtractor.java#L34-L37

Reply:
In this case being the spec?

Reply:
here it would be `// copied from DbIncubatingAttributes`